lib/compare-file-size.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const getFileSizesPerType = require('./get-filesizes-per-type');
+const promisify = require('bluebird').promisify;
+const glob = promisify(require('glob').glob);
+const path = require('path');
+
+function compareFileSize(sourceFolder, tempFolder) {
```

Comment:
I'd still refactor to `srcDir`, `destDir`, `srcFiles` and `destFiles` to be consistent. We've never used "source" or "folder" anywhere.